### ISSUE TITLE
[SearchForm][getMethods update]

### ISSUE THREAD

**Issue by atherdon** [GroceriStar/groceristar-fetch#35]
replace allergy, cuisine, course, holidays, diets methods to attribute methods
So instead of having `getAllergy()` we'll have `getAttribute('allergy')`

Tell me if you'll need more details

**Comment by atherdon** [comment]
saw this notification?

**Comment by atherdon** [comment]
and use pull requests when you'll make a changes, not a direct commits like before

**Comment by MinkoJ2** [comment]
Yes

**Comment by atherdon** [comment]
@vadim9999 can you help us with this task?

**Comment by sanchit94** [comment]
Super helpful!! :)

**Comment by atherdon** [comment]
@sanchit94 yeah, it's evolving a bit step by step. more interesting and cool features coming!

**Comment by atherdon** [comment]
updates were published.

**Comment by atherdon** [comment]
@vadim9999 please update the readme file, because it's a bit confusing right now.
we should have a getAttribute in example, but we have an old version of methods

**Comment by vadim9999** [comment]
@atherdon it's second commit in pull request
https://github.com/GroceriStar/groceristar-fetch/pull/55

**Comment by atherdon** [comment]
yeah i saw it. btw, you can meention issue number at commit and when i'll
approve request - it'll closes automatically. read more here:
https://stackoverflow.com/questions/12235620/automatically-closing-issue-from-pull-request-in-github


On 30-Jul-2018 at 10:36 AM, Vadim wrote:


@atherdon <https://github.com/atherdon> it's second commit in pull request
#55 <https://github.com/GroceriStar/groceristar-fetch/pull/55>

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/GroceriStar/groceristar-fetch/issues/35#issuecomment-408820837>,
or mute the thread
